trie/iterator.go

```diff
@@ -858,8 +859,9 @@ func zkMerkleTreeNodeBlobFunctions(findBlobByHash func(key []byte) ([]byte, erro
 			case *zk.LeafNode:
 				return &merkleTreeIteratorLeafNode{
 					hash: hash,
-					blob: n.Data(),
-					key:  zkt.ReverseByteOrder(n.Key),
+					blob: blob,
+					data: n.Data(),
```

Comment:
Just question, is this also required to resolve the related issue?

Reply:
This is unrelated to the above issue, but should be reflected in snap sync. Should I exclude it?